src/js_realm_object.hpp

```diff
@@ -150,11 +152,10 @@ typename T::Object RealmObjectClass<T>::create_instance(ContextType ctx, realm::
         if (!delegate || !delegate->m_constructors.count(name)) {
             return create_instance_by_schema<T, RealmObjectClass<T>>(ctx, schema, internal);
         }
-
-        FunctionType constructor = delegate->m_constructors.at(name);
-        auto object = create_instance_by_schema<T, RealmObjectClass<T>>(ctx, constructor, schema, internal);
-
-        return object;
+        else {
+            FunctionType constructor = delegate->m_constructors.at(name);
+            return create_instance_by_schema<T, RealmObjectClass<T>>(ctx, constructor, schema, internal);
+        }
```

Comment:
```suggestion
        FunctionType constructor = delegate->m_constructors.at(name);
        return create_instance_by_schema<T, RealmObjectClass<T>>(ctx, constructor, schema, internal);
```

Reply:
I'd like to keep this the way it is to avoid having to resolve this when rebasing `v11`.